core-spi-test/src/main/java/org/ehcache/internal/store/StoreValueHolderHitRateTest.java

```diff
@@ -41,11 +41,10 @@ public StoreValueHolderHitRateTest(final StoreFactory<K, V> factory) {
   }
 
   @SPITest
-  @Ignore(reason = "Failing test to fix")
   public void hitRateCanBeReturned()
       throws IllegalAccessException, InstantiationException {
     Store.ValueHolder<V> valueHolder = factory.newValueHolder(factory.createValue(1));
 
-    assertThat(valueHolder.hitRate(TimeUnit.MILLISECONDS), is(notNullValue()));
+    assertThat(valueHolder.hitRate(TimeUnit.MILLISECONDS.convert(Time.time(), TimeUnit.MILLISECONDS), TimeUnit.MILLISECONDS), is(notNullValue()));
```

Comment:
Why that `Time.time()` usage instead of a simpler `System.currentTimeMillis()`? Especially as the former actually returns `System.nanoTime()`.